sdk/storage/azure-storage-queue/azure/storage/queue/_queue_client.py

```diff
@@ -445,11 +442,12 @@ def set_queue_access_policy(self,
 
     @distributed_trace
     def send_message(
-            self,
-            content,  # type: Any
-            **kwargs  # type: Any
-    ):
-        # type: (...) -> QueueMessage
+        self, content: Any,
```

Comment:
nit: Can you update the docstring for `content` below to be `Any` instead of `obj`?

Same below on `update_message` and in `async`.

Reply:
Drats. SO CLOSE! But good catch 😄